bin/tap-difflet

```diff
@@ -31,22 +90,23 @@ var timer   = hirestime();
 var errors  = [];
 
 tap.on('comment', function(comment) {
-    if (/^tests\s+[0-9]+$/gi.test(comment)) {
-        output('\n');
-        comment = chalk.white.bold(comment);
-    }
-    else if (/^pass\s+[0-9]+$/gi.test(comment)) {
-        comment = chalk.green.bold(comment);
-    }
-    else if (/^fail\s+[0-9]+$/gi.test(comment)) {
-        comment = chalk.red.bold(comment);
-    }
-    else if (/^ok$/gi.test(comment)) {
-        return;
-    }
-    else {
-        output('\n');
-        comment = chalk.white.bold(comment);
+  
+    switch (commentType(comment)) {
```

Comment:
please do not use `switch` and stick with `if` Nice to extract out the `commentType` (but could you rename it to `getCommentType`.